dll/win32/ws2_32/src/sockctrl.c

```diff
@@ -122,6 +122,18 @@ INT
 WSAAPI
 listen(IN SOCKET s,
        IN INT backlog)
+{
+	//make a saperate thread call listen_call()
+	return WSACallThread( 1,(LPVOID) s, (LPVOID) backlog);
+}
```

Comment:
Do we put a space between casting variables? Can someone clarify if this is a coding style violation?

Reply:
It is my real name  (the last two characters are the first two characters of my last name). I changed my public profile and entered my full name + my e-mail address, I hope it's correct this way?

Reply:
I deleted the space :)

Reply:
@Fraizeraust : normally our code style looks like: `return WSACallThread(1, (LPVOID)s, (LPVOID)backlog);`.

Reply:
Your commits use your full name, so that looks good.

Hoi btw :)

Reply:
4-space indentation please :)